tests/failing

```diff
@@ -115,3 +115,13 @@ tests/nofib-exe-examples/lastpiece.uplc
 tests/nofib-exe-examples/prime-P200.uplc
 tests/nofib-exe-examples/primetest-large.uplc
 tests/nofib-exe-examples/queens-6-bt.uplc
+#
+# Error examples
+#
+# Programs in addInteger-one-arg.uplc and addInteger-no-arg.uplc call
+# addInteger with a single argument or none. In the current semantics,
+# kplc returns an error. However, they are valid programs as
+# that should return a term. This of course should be repetead for the
+# remaining builtins and will not be included until we solve this.
+tests/error/addInteger-one-arg.uplc
+tests/error/addInteger-no-arg.uplc
```

Comment:
I think these need to be added to `test/simple` because they aren't supposed to produce an error